### ISSUE TITLE
For the replay preview in Issue Details, HMW show a more relevant video snippet

### ISSUE THREAD

**Issue by jas-kas** [getsentry/sentry#38462]
**Problem:**

Today, we show the entire session as the replay preview in the Issue Details for an error event and the timestamp is always at the beginning of the session, regardless of when the error occurred.

For example, there may be a 20m replay preview that spans across 8 different errors that always starts at the beginning of the session, and we do not bring the user to the error event that they are viewing the details for... making it difficult to gain value from this replay preview without opening up the Replay Details.

HMW connect the replay video to the error event on that view?

**Solution:**

TBD on Design.

**Comment by ryan953** [comment]
We're going to start the replay near the timestamp of the specific error event.

**Comment by ryan953** [comment]
We've clarified this other ticket: https://github.com/getsentry/sentry/issues/33887. That says that on the Issue Details page we will start the replay at the timestamp when the error happened.

Given this change, i think ticket is mostly focused on making the ui emphasis that this is a snippet of the replay. Also this ticket touches on data loading. Both of these seem really hard and out of scope for the external alpha milestone.

**Comment by Jesse-Box** [comment]
@ryan953 I don't think we need to do a snippet per say, rather we just fast forward the replay to (just as an example) 5 secs before the error. The idea is just so they can hit play and see the error they're seeing in the breadcrumb.

Does that make things easier?

**Comment by ryan953** [comment]
> I don't think we need to do a snippet per say, rather we just fast forward the replay to (just as an example) 5 secs before the error. The idea is just so they can hit play and see the error they're seeing in the breadcrumb.

@Jesse-Box This is implemented now! On the Issues Details page we scrub the video to the exact spot when the error happened. We've got the "go back 10s" button, so i think it's easy for people to get to an interesting spot from there.  


With that done, I don't think there's anything captured in this ticket that is outstanding. Closing as complete / duplicate of #33887
